Contentless/Program.cs

```diff
@@ -43,11 +44,62 @@ public static class Program {
         var excluded = config.ExcludedFiles.Select(Program.MakeFileRegex).ToArray();
         var overrides = Program.GetOverrides(config.Overrides).ToArray();
 
+        var referencesVersions = config.References.ToList().ToDictionary(x => x, x => (string)null, StringComparer.OrdinalIgnoreCase);
+        if (config.References.Any())
+        {
+            var csprojPath = args[1];
+            Console.WriteLine($"Using project file {csprojPath}");
+            var projectRootElement = ProjectRootElement.Open(csprojPath);
+            foreach (var property in projectRootElement.AllChildren.Where(x => x.ElementName == "PackageReference").Select(x => x as ProjectItemElement))
+            {
+                var libraryName = property.Include;
+                var version = (property.Children.First() as ProjectMetadataElement).Value;
```

Comment:
`First` and `as` should be replaced with `FirstOrDefault` and an `is` or a null check - unless there's a reason neither of these can be null that I'm not aware of?